src/sql/src/plan/statement.rs

```diff
@@ -1543,6 +1547,69 @@ fn handle_create_source(scx: &StatementContext, stmt: Statement) -> Result<Plan,
     }
 }
 
+fn handle_create_table(scx: &StatementContext, stmt: Statement) -> Result<Plan, anyhow::Error> {
+    match &stmt {
+        Statement::CreateTable {
+            name,
+            columns,
+            constraints,
+            with_options,
+            if_not_exists,
+        } => {
+            if !with_options.is_empty() {
+                unsupported!("WITH options");
+            }
+            if !constraints.is_empty() {
+                unsupported!("CREATE TABLE with constraints")
+            }
+
+            let names: Vec<_> = columns
+                .iter()
+                .map(|c| Some(normalize::column_name(c.name.clone())))
+                .collect();
+
+            // Build initial relation type that handles declared data types
+            // and NOT NULL constraints.
+            let typ = RelationType::new(
+                columns
+                    .iter()
+                    .map(|c| {
+                        let ty = scalar_type_from_sql(&c.data_type)?;
+                        let nullable = !c.options.iter().any(|o| o.option == ColumnOption::NotNull);
+                        Ok(ColumnType::new(ty).nullable(nullable))
+                    })
+                    .collect::<Result<Vec<_>, anyhow::Error>>()?,
+            );
+
+            for (_, column) in columns.iter().enumerate() {
+                for option in column.options.iter() {
+                    if let ColumnOption::Unique { is_primary } = option.option {
+                        if is_primary {
+                            unsupported!("CREATE TABLE with primary keys");
+                        }
```

Comment:
This should reject all `ColumnOption`s besides `ColumnOption::NotNull`. Unique keys are going to be problematic. (Are there other column options? If so they will also be problematic.)